packages/rule-no-p3p/src/no-p3p.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @fileoverview Don't use P3P related headers
+ */
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+// The list of types depends on the events you want to capture.
+import { IAsyncHTMLElement, FetchEnd, Response, IRule, RuleMetadata } from 'sonarwhal/dist/src/lib/types';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+import { getIncludedHeaders } from 'sonarwhal/dist/src/lib/utils/rule-helpers';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class NoP3pRule implements IRule {
+
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.development,
+            description: `Don't use P3P related headers or meta tags`
+        },
+        id: 'no-p3p',
+        schema: [
+            /*
+             * If you want to allow the user to configure your rule
+             * you should use a valid JSON schema. More info in:
+             * https://sonarwhal.com/docs/contributor-guide/rules/#themetaproperty
+             */
+        ],
+        scope: RuleScope.any
```

Comment:
Does this rule make sense with local files? if not, chante the scope to `RuleScope.site`.